knowledge-base/print-report-correctly-from-winforms-viewer-without-declaring-entire-application-as-dpi-aware.md

```diff
@@ -0,0 +1,39 @@
+---
+title: How to display/print report correctly from Windows Forms Viewer without declaring the entire application as DPI-aware
+description: What can we do to print and display Telerik reports correctly in WinForms application that should not be declared as DPI-aware
+type: how-to
+page_title: Display/print report correctly from Windows Forms Viewer without declaring the entire application as DPI-aware
+slug: print-report-correctly-from-winforms-viewer-without-declaring-entire-application-as-dpi-aware
+position: 
+tags: 
+ticketid: 1425131
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Progress® Telerik® Reporting</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Description
+When the Windows Forms Viewer is used in a WinForms application that is not DPI-aware, when printing or previewing a report it may be incorrectly scaled.
+
+The recommended solution is to set the application to DPI-aware as explained in [Declaring the application as DPI-aware](../winforms-report-viewer#declaring-the-application-as-dpi-aware) article section.
+However, this may result in the distortion of other controls in the application.
+
+## Solution
+
+1. You may extract the viewer in a separate application, and call the latter when needed from the main application. Only the application with the viewer (with embedded Reporting engine) should be declared as DPI-aware.  
+  
+2. You may move the Reporting engine in a separate [Telerik Reporting REST Service](../telerik-reporting-rest-conception) and [use the Windows Forms Viewer with the REST Service](../winforms-viewer-howto-use-it-with-rest-service). It will be necessary to add the <**dpiAware**\> section to the REST Service application's **web.config** and set the **_dpiAwareness_** attribute to *PROCESS\_SYSTEM\_DPI\_AWARE* - [dpiAware element](../configuring-telerik-reporting-dpiaware#dpi-awareness-in-report-viewers).  
+  
+3. You may test to run the application not declared as DPI-aware on a machine with Windows 10 Anniversary Update, where Windows runtime has improved the UX for Windows Forms applications that are not declared as DPI-aware. We cannot guarantee the report will be displayed as expected.  
+  
+4. Another option (*not recommended*) is to set the 'OutputFormat' attribute of [ImageInteractive Device Information Settings](../device-information-settings-imageinteractive) to 'PNG' (the default is 'EMF'). In this case the zooming will not work well, as the report will be rendered as raster image (as opposed to vector image). This setting will take effect for reports displayed in _Interactive_ mode in the Windows Forms Report Viewer.  
```

Comment:
I would avoid using "not recommended" here, but rather describe it like "this approach is more suitable for printing the document rather than previewing it due to raster zoom" or something like this.